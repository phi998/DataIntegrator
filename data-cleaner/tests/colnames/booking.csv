Name,Description,Rating,Benefits,Name,Reviews,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Location,Description,Price,Rating,Rating,Rating,Rating,Rating,Benefits,Benefits,Rating,Benefits,Name,Price,Price,Rating,Rating,Price,Price,Location,Rating,Price,Location,Rating,Price,Rating,Price,Price,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Price,Rating,Unknown,Rating,Location,Description,Name,Location,Description,Button,Rating,Unknown,Benefits,Name,Name,Action,Name,Button,Benefits,Description,Description,Benefits,Availability,Benefits,Description,Description,Benefits,Benefits,Age Restriction,Age Requirement,Description,Description,Benefits,Benefits,Description,Benefits,Name,Rating,Location,Price,Name,Rating,Location,Price,Name,Rating,Price,Name,Rating,Location,Price,Description,Location,other,Name,Location,Benefits,Benefits,Rating,Benefits,Benefits,Benefits,Name,Number of Max Persons,Price,Price,Benefits,Benefits,Benefits,Description,Price,Name,Price,Price,Unknown,Location,Location,Distance,Rating,Benefits,Name,Rating,Location,Price,Name,Rating,Location,Price,Name,Rating,Location,Price,Rating,Price,Rating,Description
Name,Description,Rating,Actions,Name,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Benefits,Actions,Price,Rating,Rating,Rating,Rating,Rating,Benefits,Labels,Rating,Benefits,Actions,Rating,Price,Rating,Rating,Price,Price,Price,Rating,Rating,Benefits,Rating,Price,Price,Price,Price,Rating,Rating,Rating,Price,Rating,Rating,Price,Price,Rating,Description,Rating,Rating,Unknown,Actions,City,Description,actions,Rating,Actions,Benefits,Actions,Name,Actions,Name,Actions,Benefits,Actions,Description,Check in hours,Check in hours,Description,Description,Unknown,Description,Age Range,Check in hours,Check in hours,Description,Check in hours,Benefits,Actions,Description,Actions,Name,Rating,Distance from center,Price,Name,Rating,Distance from center,Price,Name,Rating,Price,Name,Rating,Distance from center,Price,Description,Actions,Labels,Name,City,Benefits,Benefits,Benefits,Benefits,Parking,Actions,Name,Max Person,Price,Price,Benefits,Other,Actions,Benefits,Price,Description,Price,Price,Unknown,Name,Name,Distance,Rating,Benefits,Name,Rating,Distance from city center,Price,Name,Rating,Distance from city center,Price,Name,Rating,Distance from center,Price,Name,Price,Rating,Description
