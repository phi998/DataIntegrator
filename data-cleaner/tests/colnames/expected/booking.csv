Name,.,Rating,.,Name,Reviews,Rating,Rating,Rating,Rating,Rating,Rating,Rating,.,.,.,.,.,.,.,.,Benefits,.,Rating,Benefits,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,Description,Rating,Location,.,.,Location,.,.,Reviews,.,.,.,.,Action,Name,.,Benefits,.,Name,Check in hours,Check in hours,.,.,.,Benefits,.,Age Restriction,Age Requirement,.,.,.,.,.,.,Name,Rating,Location,Price,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,Location,Location,Distance,.,Benefits,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,Description
